Libraries/Components/WebView/WebView.android.js

```diff
@@ -127,13 +134,25 @@ var WebView = React.createClass({
       domStorageEnabled = this.props.domStorageEnabledAndroid;
     }
 
+    var htmlProp, urlProp, sourceProp;
+    if(this.props.html) {
+      console.warn('html is deprecated. Use source instead.');
+      htmlProp = this.props.html;
+    } else if(this.props.url) {
+      console.warn('url is deprecated. Use source instead.');
```

Comment:
There doesn't seem to be anything in react to do this. Something like https://github.com/react-bootstrap/react-prop-types/blob/master/src/deprecated.js would be nice, it could eventually be integrated with the website too.

If you like I can submit a PR that implements that and change the places we deprecate props manually.